platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/pax/web/jetty/FilterInjector.java

```diff
@@ -58,8 +61,12 @@
    * @param filterList filters that should be injected.
    * @param executorService used to check for missed servlet contexts
    */
-  public FilterInjector(List<Filter> filterList, ScheduledExecutorService executorService) {
+  public FilterInjector(
```

Comment:
✏️ Could update this method to just take in referenceFilterList and update blueprint to create services for other filters but I'm okay with how it is.

Reply:
Seemed like it introduced unnecessary back and forth between blueprint and this bundle so I didn't do it.